content/en.json

```diff
@@ -92,13 +88,13 @@
       "subtitle": "We work in a modern way, aiming for quality. \"First time right\" is also cheapest in the end.",
       "how-title": "How we like to do this:",
       "how": [
-        "Modern version control",
-        "Up to date tooling, frameworks, and libraries",
+        "Everything in version control",
```

Comment:
The original intent around "modern" was because all too many folks are still stuck on SVN or TFSVC, although the updated text is also nice, so I'm good with either way. (Just wanted to mention that "modern" wasn't randomly chosen as a term.)

Reply:
Sure, I personally don't think of "git" as modern (and maybe I've become a bit allergic to the use of "modern" because it is often used as an empty buzzword -- but I think it's use to distinguish git from older, worse version constrol systems is fair).